### ISSUE TITLE
SSL certificate for the API is invalid

### ISSUE THREAD

**Issue by CyberLizard** [blockstrap/framework#50]
Hi, your certificate for api.blockstrap expired on 10/11/2015 which causes a number of problems. Is there an ETA when this will be resolved or is the project dead? http://prntscr.com/8tc2l3

**Comment by wdl1908** [comment]
Any progress on this?

**Comment by CyberLizard** [comment]
Yes a newsletter send out a good 2 weeks ago suggesting to change the code to non-SSL which obviously doesn't work for a number of reason. It's sad to see that Blockstrap has been pretty much rendered useless over a few $ to renew a SSL certificate.

**Comment by JohnnyMayo** [comment]
Hey all - sorry for not getting back to you sooner on this. While it's no excuse, we're still only a team of two developers - one of which just welcomed his second child. Blockstrap was taking up an increasing amount of time and resources to the point where it was becoming unsustainable for us to manage, and having survived without funding for a year longer than we had planned, we had to make the difficult decision of switching the API off. This was our baby for the last 18 months so it really wasn't an easy call for us to make. We're still actively working on bitcoin projects and are focusing the remaining funds and strength we have into our new protocols. Should we succeed and survive, we'll almost certainly come back for Blockstrap. We had an amazing time doing it. 

I want to thank you guys especially for your support over the last 18 months. If you'd like to know more about what we're up to now, feel free to drop me a message directly. When we get future news on Blockstrap I'll be sure to let you know.
